### ISSUE TITLE
WhaleySans font

### ISSUE THREAD

**Issue by whaleygeek** [carlosperate/awesome-microbit#619]
I've had a lot of interest in this font since we aired the Blue Peter escape room challenge. I looked and can't see it on here, perhaps you could add it?

The original font was in MicroPython from here:
https://github.com/whaleygeek/mb_clock

Another user converted it into a MakeCode package for me here:
https://github.com/makecode-extensions/WhaleySansFont

**Comment by carlosperate** [comment]
Yeah, that'd be a great addition! Would you like to create a PR to add them?
